src/main/java/racing/view/ResultView.java

```diff
@@ -1,20 +1,27 @@
-package ui;
+package racing.view;
 
-import racing.Car;
-import racing.RacingCars;
+import racing.domain.Car;
+import racing.domain.RacingCars;
+import racing.domain.RacingHistory;
 
 import java.util.Iterator;
 import java.util.Set;
 
 public class ResultView {
 
-    public static void showRaceResult(RacingCars racingCars, int tryCount) {
-        if (tryCount == 0) {
+    public static void showRaceResult(Set<RacingHistory> racingHistory) {
+        Iterator<RacingHistory> historyIterator = racingHistory.iterator();
+        while (historyIterator.hasNext()) {
+            RacingHistory nextRound = historyIterator.next();
+            showRaceRound(nextRound.getRound(), nextRound.getHistorySet());
+        }
+    }
+
+    private static void showRaceRound(Integer round, Set<Car> racingCarsSet) {
+        if (round == 1) {
```

Comment:
```suggestion
        if (RacingHistory.isFirst()) {
```

저는 객체에게 물어볼것 같아요.
이번 과정속에서는 과도할정도로 객체를만들고 과도할정도로 객체에게 메시지를 보내는 훈련을 의식적으로 해보시면 어떨까요? ㅎ